packages/flutter/test/widgets/localizations_test.dart

```diff
@@ -53,12 +51,12 @@ class FakeLocalizationsDelegate extends LocalizationsDelegate<String> {
 
 class TestAutomatedTestWidgetsFlutterBinding extends AutomatedTestWidgetsFlutterBinding {
 
-  VoidCallback onAllowFrame;
+  VoidCallback? onAllowFrame;
 
   @override
   void allowFirstFrame() {
     if (onAllowFrame != null)
-      onAllowFrame();
+      onAllowFrame!();
```

Comment:
This and the line before it could be replaced with:
`onAllowFrame?.call();`

Reply:
Couldn't we just remove the `if` statement here since `onAllowFrame?.call()` effectively checks if onAllowFrame is `null` anyway?

Reply:
Yes, that's what I intended.